cmd/krel/cmd/release_notes.go

```diff
@@ -80,21 +84,28 @@ permissions to your fork of k/sig-release and k-sigs/release-notes.`,
 		github.TokenEnvKey),
 	SilenceUsage:  true,
 	SilenceErrors: true,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		// If none of the operation modes is defined, show the usage help and exit
+		if !releaseNotesOpts.createDraftPR && !releaseNotesOpts.createWebsitePR {
+			if err := cmd.Help(); err != nil {
+				logrus.Error(err)
+			}
+		}
+	},
 	RunE: func(cmd *cobra.Command, args []string) error {
+		// Run the PR creation function
 		return runReleaseNotes()
 	},
 }
 
 type releaseNotesOptions struct {
 	tag             string
-	draftOrg        string
+	GithubOrg       string
```

Comment:
What is the reason to have this one public?
```suggestion
	githubOrg       string
```